pyproject.toml

```diff
@@ -19,7 +19,8 @@ mysql = [
     "apache-airflow[mysql]"
 ]
 postgres = [
-    "apache-airflow[postgres]"
+    "apache-airflow[postgres]",
+    "psycopg2"
```

Comment:
Though `psycopg2` is not imported by airflowHPC internally, I think it might still be good to add `psycopg2` as an optional dependency, since people using a PosgreSQL database will need `psycopg2` anyway when running `export AIRFLOW__DATABASE__SQL_ALCHEMY_CONN="postgresql+psycopg2://airflow_user:airflow_pass@localhost/airflow_db"` as instructed by README. With that being said, feel free to remove this if you think that is more reasonable!

Reply:
I think this might actually be an issue in airflow if you got an error about this. Installing `airflow[postgres]` has the dependency `"psycopg2-binary>=2.8.0"` so there should not be an issue. Did you get an error about this?

Reply:
So I installed airflow in two different places (my lab desktop and Bridges-2), `AIRFLOW__DATABASE__SQL_ALCHEMY_CONN="postgresql+psycopg2://airflow_user ...` led to an error like `No module named psycopg2` in both cases. On my lab desktop, I was not able to do `pip3.10 install psycopg2`, but `pip3.10 install psycopg2-binary` worked and solved the issue. On Bridges-2, simply trying `pip3.10 install psycopg2` solved the issue, so I thought having `psycopg2` as an optional dependency could be convenient. 

> Installing airflow[postgres] has the dependency "psycopg2-binary>=2.8.0"
I could not find `psycopg2` defined as an optional dependency in `pyproject.toml` in Apache-airflow (https://github.com/apache/airflow/blob/main/pyproject.toml), or maybe I looked at the wrong file?

Reply:
Yes, it is only `psycopg2-binary` that is a dependency for airflow with postgre. If you try to do `pip install pyscopg2` without first installing the binary version, you will get an error from pip about it not finding `pg_config`, whereas the binary "just works" and also allows `pip install pyscopg2` to work. Thus I am a bit hesitant to include psycopg2 as a dependency.

Reply:
Or should we include `psycopg2-binary` instead of `psycopg2` as an optional dependency? In my case, it seemed that `pyscopg2-binary` was not installed even after `pip install airflowHPC[postgres]`, so I initially thought that `pyscopg2`/`pyscopg2-binary` was not listed as a dependency for Apache airflow. 

Anyway, we can refrain from including `pyscopg2` and just add a note to README to remind people, whichever makes more sense. (Though this might be trivial to the users anyway since they could probably easily figure out that they might need to install `psycopg2`/`psycopg2-binary` for the command to work).

Reply:
If you ran into the issue then it is probably a good idea to make `psycopg2-binary` a dependency.

Reply:
Okay added!